core_backend/tests/rails/test_language_indentification.py

```diff
@@ -0,0 +1,41 @@
+from typing import List, Tuple
+
+import pytest
+import yaml
+
+from core_backend.app.configs.llm_prompts import IdentifiedLanguage
+from core_backend.app.llm_call.parse_input import identify_language
+from core_backend.app.schemas import UserQueryRefined
+
+pytestmark = pytest.mark.rails
+
+
+LANGUAGE_FILE = "tests/rails/data/language_identification.yaml"
```

Comment:
What pytest args are you using btw?

We are importing `core_backend` so I had to run the tests within project root. But then this path wouldn't work, so I had to prepend `core_backend/` to the path.

Reply:
I ran 
```
cd aaq-core
python -m pytest -rPQ ./core_backend/tests/rails
```

Reply:
Ah i just go to core_backend and run 

`pytest -m rails` for this one.